frontend/packages/core/src/paper.tsx

```diff
@@ -14,6 +14,8 @@ const StyledPaper = styled(MuiPaper)({
   minHeight: "inherit",
 });
 
-const Paper = ({ children }: PaperProps) => <StyledPaper>{children}</StyledPaper>;
+const Paper = ({ children, ...props }: PaperProps) => (
+  <StyledPaper {...props}>{children}</StyledPaper>
+);
```

Comment:
Not honestly sure why this is necessary but it would allow me to override the padding using styled components without it. Originally I had added this to use the `sx` prop but it wound up working for this use case so I left it.